### ISSUE TITLE
Prototype a phase-harmonic audiovisual fit probe from human cockpit trajectories

### ISSUE THREAD

**Issue by rhythmatician** [rhythmatician/FractalSync#126]
Part of #123 and research branch of #77.

Coordinates with #124 (intrinsic visual gesture substrate), #121 (manual Controls-v2 trajectory capture), #111 (cockpit/replay telemetry), #92/#100 (canonical CycleBank state), #109 (AudioActivation), and #104 (later PlayerPolicy work).

- **Blocked by:** #124 for canonical visual gesture streams
- **Needs data capture from:** #121/#111
- **Does not block:** continued Physics/cockpit work
- **Blocks:** any claim that phase-harmonic entrainment should become a training reward

## Goal

Run a deliberately small, falsifiable experiment before training a Player against the audiovisual-entrainment theory.

The question is:

> **Can compact phase-aware statistics of canonical audio oscillations and intrinsic Mandelbrot motion distinguish a genuinely matched human-driven music/trajectory pair from plausible mismatches on held-out performances?**

The first probe should use no pixels, no semantic emotion labels, no absolute Mandelbrot locations, and no large neural network. If the proposed common representation is real, a small statistical or linear model should detect at least some of it.

A negative result is a valid outcome. Do not rescue the hypothesis by adding arbitrary features until the primary controls and failure analysis are complete.

## Primary data unit

A sample is a synchronized, canonical-time excerpt containing:

```text
Audio side
  canonical CycleBank / causal analytic audio state

Visual side
  recorded Controls-v2 human cockpit trajectory
  authoritative manifold Physics snapshots
  #124 intrinsic visual gesture stream

Metadata
  song/performance identity
  run identity
  reset/initial state
  Physics/Map/contract versions
  exact sample-time alignment
```

Capture several human runs per excerpt where practical. A run need not be expert or globally beautiful, but the driver should mark whether the attempt felt musically matched, neutral, or poor. Preserve rejected/awkward runs rather than retaining only successes.

### Audio

Start with canonical phase-bearing evidence already available from CycleBank:

```text
a_i(t) = A_i(t) exp(i phi_i(t))
```

including strength/confidence and continuous frequency. Optionally add a small causal complex-Morlet bank over justified AudioActivation channels only as a separately identified ablation.

Do not rebuild CycleBank in Python.

### Visual

Consume the Rust-authoritative #124 streams. Compare at least:

1. direct intrinsic motion phasor

   ```text
   m_parallel(t) = u(t) exp(i Theta(t));
   ```

2. causal complex-Morlet coefficients of selected real intrinsic gesture channels:

   ```text
   u
   Omega
   a_parallel
   a_perp
   eta_M
   eta_M_dot
   ```

3. modulus/amplitude-only versions of the same channels as controls.

`kappa_g` may be included only with the #124 low-speed validity gate. Do not let division by small `u` create a spurious high-value feature.

The primary experiment must exclude:

- raw absolute `x,y` or `c`;
- camera coordinates or screen-space heading;
- Julia RGB pixels;
- song identity;
- manually entered transition labels;
- future audio in any feature claimed to be runtime-causal.

Coordinate-specific features may appear only in a diagnostic shortcut baseline whose purpose is to detect leakage, not to support the main claim.

## Phase-harmonic statistics

For a nonzero complex coefficient

```text
z = |z| exp(i arg z),
```

define

```text
[z]^k = |z| exp(i k arg z),
```

and `[0]^k = 0`.

For audio coefficient `a_i` and visual coefficient `b_j`, estimate causal/windowed cross-moments

```text
C_ij^(n,m)(s,r,tau)
  = E_t(
      [a_i(s,t)]^n
      * conjugate([b_j(r,t-tau)]^m)
    ).
```

Use a normalized statistic such as

```text
gamma_ij^(n,m)
  = C_ij^(n,m)
    / sqrt(
        E|[a_i]^n|^2
        * E|[b_j]^m|^2
        + epsilon
      ).
```

Initial search budget:

```text
n,m in {1,2,3}
small documented audio/visual scale banks
small signed lead/lag grid
several causal window lengths
```

Keep this grid small enough to enumerate, inspect, and correct for multiple comparisons. Do not turn unconstrained hyperparameter search into evidence.

Carry amplitude, confidence, low-speed validity, stream epoch, and missing-mode masks into every statistic.

## Pair construction

For each positive aligned pair, construct several negative/control families.

### 1. Large within-performance time shifts

Pair the trajectory with the same audio shifted by amounts large enough to disrupt local phase/phrase alignment. Exclude trivial wraparound overlap and record the exact shift.

### 2. Local phase-disrupting shifts

Use smaller shifts measured relative to active CycleBank periods. This tests whether the statistic is detecting phase fit rather than only song identity or global envelope.

### 3. Wrong-performance hard negatives

Pair with another performance/excerpt chosen to resemble the positive in duration and coarse energy/tempo distribution where possible.

### 4. Same-excerpt alternate runs

Compare a human-rated good run with neutral/poor runs over the same audio and reset family. This is especially important because song identity, tempo, and envelope are held constant.

### 5. Phase-randomized controls

Preserve coefficient magnitudes approximately while randomizing or scrambling phase in a documented way. This tests the incremental value of phase-bearing structure.

### 6. Gesture-channel permutation

Shuffle visual gesture channels or independently offset them to test whether a generic amount of oscillation is sufficient.

Do not treat one easy negative family as decisive. A representation that only separates different songs but cannot distinguish aligned from shifted same-song runs has not established audiovisual fit.

## Dataset splitting and leakage controls

All model selection and evaluation must be grouped so overlapping windows from one underlying run never cross train/validation/test boundaries.

Report at least:

```text
leave-one-song/performance-out
leave-one-human-run-out
same-excerpt good-vs-poor evaluation
```

Where the corpus is too small for a stable numeric claim, report uncertainty honestly and collect more cockpit runs rather than allowing overlapping-window pseudo-replication.

Mechanically test that:

- normalization statistics are fit on training folds only;
- no future samples enter causal features;
- no song/run/reset identifier leaks into the probe;
- positive and negative examples have matched window-count treatment;
- time shifts cannot be identified from padding, epoch boundaries, or missing-data artifacts.

## Probe models

Prefer simple, auditable probes:

```text
regularized logistic regression
linear ranking model
linear discriminant / ridge probe
small canonical-correlation or SVD analysis
```

A tiny nonlinear baseline may be included after the linear result, but no large temporal network is permitted in the primary experiment. The purpose is to test the representation, not whether a powerful classifier can memorize the dataset.

## Required baselines

Compare the phase-harmonic representation against at least:

1. **Chance / label permutation.**
2. **Audio and visual amplitude/modulus only.**
3. **Coarse summary baseline:** audio energy/rate statistics versus visual `K/u` and total gesture energy.
4. **Ordinary zero/lagged correlation or conventional coherence** without phase harmonics.
5. **Direct `1:1` phase locking only.**
6. **Phase-randomized coefficients.**
7. **Coordinate shortcut diagnostic:** raw `c/x/y` and screen heading, clearly excluded from the claimed intrinsic result.

The central question is not merely whether phase harmonics work. It is whether they add stable held-out information beyond simpler amplitude and correlation summaries.

## Human-fit labels

Collect a small blinded ranking/ordinal dataset where a reviewer judges synchronized audio/trajectory clips without seeing the feature values or model score.

At minimum compare:

- true aligned run;
- shifted version of the same run;
- alternate human run over the same excerpt;
- wrong-performance hard negative.

Record uncertainty/ties. Do not force a total ordering when the reviewer cannot distinguish clips.

Evaluate whether the probe score:

- ranks the true pair above mismatches;
- distinguishes good versus poor same-excerpt runs;
- correlates with held-out human fit judgments;
- fails gracefully on genuinely ambiguous pairs.

Automated matched-pair classification alone is insufficient if its score does not correspond to human-perceived fit.

## Low-dimensional coupling test

At each temporal scale or context window, estimate a whitened cross-covariance/canonical-coherence operator between audio and visual phase-harmonic feature groups.

Inspect its singular spectrum and held-out stability:

```text
C = S_VV^(-1/2) S_VA S_AA^(-1/2)
C = U Sigma V^H.
```

Questions:

- Do one to a few singular modes dominate?
- Are their audio/visual loadings stable under bootstrap and song-held-out refits?
- Do they survive amplitude-only and phase-randomized controls?
- Do they correspond to interpretable gesture combinations without depending on absolute `c`?

Do not claim a low-dimensional choreography space solely from an in-sample singular-value plot.

## Pre-registration / decision rule

Before inspecting final held-out results, write a short checked-in experiment plan specifying:

- exact excerpts/runs and grouped splits;
- positive/negative construction;
- channel set;
- `n:m`, scale, lag, and window grid;
- primary probe and regularization selection;
- primary metrics;
- bootstrap/permutation procedure;
- what result would justify opening/unblocking a reward prototype;
- what result would count as inconclusive or negative.

The exact numeric threshold should follow dataset size and uncertainty analysis, but it must be fixed before the final test fold is examined.

A reasonable qualitative gate is:

> phase-aware intrinsic features must show repeatable song/performance-held-out improvement over amplitude-only and ordinary-correlation baselines, and the resulting score must agree directionally with independent human fit judgments.

## Outputs

Produce:

- a versioned trajectory/audio extraction manifest;
- reusable offline feature/statistics code consuming Rust bindings;
- fold-safe positive/negative dataset construction;
- baseline and phase-harmonic comparison tables;
- bootstrap/permutation uncertainty;
- singular-spectrum/stability diagnostics;
- blinded human-rating comparison;
- several inspectable success and failure examples linked to #111 replay artifacts;
- an explicit proceed / revise / reject verdict for the reward work.

## Acceptance

- [ ] Canonical human-driven cockpit trajectories are captured with exact audio/sample-time alignment and version metadata.
- [ ] #124 intrinsic gesture channels are consumed from Rust rather than recomputed independently.
- [ ] Positive, within-song shift, wrong-performance, same-excerpt alternate-run, and phase-randomized controls exist.
- [ ] Grouped splits prevent song/run/window leakage.
- [ ] A small pre-registered linear/ranking probe is evaluated on held-out performances.
- [ ] Amplitude-only, coarse summary, ordinary-correlation, direct-`1:1`, and phase-randomized baselines are reported.
- [ ] Phase-harmonic incremental value is reported with uncertainty, not only best-case accuracy.
- [ ] Human fit judgments are collected blind and compared with the automated score.
- [ ] Low-dimensional coupling claims, if any, are supported by held-out/bootstrap stability.
- [ ] Failure cases are replayable and inspected for timing, validity, low-speed, and data-leak artifacts.
- [ ] A clear verdict states whether #126 is justified, needs revision, or should remain blocked.
- [ ] No PlayerPolicy is trained and no speculative reward is promoted into the product path.

## Non-goals

- Do not train the final visualizer.
- Do not tune human cockpit behavior to maximize the probe after seeing test results.
- Do not require semantic emotion, beat, bar, or phrase labels.
- Do not use pixels in the primary experiment.
- Do not infer success from different-song classification alone.
- Do not use a large neural classifier to compensate for a weak representation.
- Do not add candidate diagnostics to PlayerObservation.
- Do not claim causality from offline correlation.
- Do not silently convert a negative result into an indefinitely expanding feature search.

**Comment by rhythmatician** [comment]
Numbering correction after two connector retries were auto-closed as duplicates: the downstream reward ticket is **#128**. In the Acceptance section, “whether #126 is justified” should be read as “whether #128 is justified.” #126 is the falsification probe itself and is the evidence gate for #128.
